homeassistant/components/irm_kmi/quality_scale.yaml

```diff
@@ -0,0 +1,29 @@
+rules:
+  # Bronze
+  action-setup:
+    status: exempt
+    comment: >
+      No service action implemented in this integration at the moment.
+  appropriate-polling:
+    status: done
+    comment: >
+      Polling interval is set to 7 minutes.
+  brands: done
+  common-modules: done
+  config-flow-test-coverage: done
+  config-flow: done
+  dependency-transparency: done
+  docs-actions:
+    status: exempt
+    comment: >
+      No service action implemented in this integration at the moment.
+  docs-high-level-description: done
+  docs-installation-instructions: done
+  docs-removal-instructions: done
+  entity-event-setup: done
+  entity-unique-id: done
+  has-entity-name: done
+  runtime-data: done
+  test-before-configure: done
+  test-before-setup: done
+  unique-config-entry: done
```

Comment:
Let's evaluate all rules

Reply:
They are now all evaluated.  `quality_scale.yaml` is update accordingly in commit 9cfc6e9e786d4373a32550a2f98c6bedcb980322

More info in comment below https://github.com/home-assistant/core/pull/144689#issuecomment-2892175369